packages/client/src/components/UnprotectedRoute.tsx

```diff
@@ -31,7 +31,9 @@ const UnprotectedRouteWrapper = (props: RouteProps) => {
           document.title = config?.APPLICATION_NAME as string
           dispatch(configAnonymousUserLoaded({ anonymousConfig: config }))
         }
-      } catch (e) {}
+      } catch (e) {
+        console.log('Error to load configuration')
+      }
```

Comment:
Could you remove the try-catch completely? A normal user wouldn't see this error in their console and wouldn't know anything is wrong